torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -133,6 +138,64 @@ def dispatch(
             diagnostic_context,
         )
 
+    @_beartype.beartype
+    @diagnostics.diagnose_call(
+        diagnostics.rules.find_operator_overloads_in_onnx_registry,
```

Comment:
@BowenBao Does it make sense to have two functions calling the same rule?

Reply:
Not really. Each op will populate two diagnostics of this same rule, but one for this function and another for `get_function_overloads`.

Reply:
Moved it down to `get_function_overloads`